omod-2.0/src/test/resources/providerTestDataset.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<dataset>
+	<provider provider_id="2" person_id="502" name="Mr. Valid Provider" identifier="Test2" creator="1" date_created="2005-01-01 00:00:00.0" retired="false" uuid="f753e10d-9409-4422-9b49-a062b2ec89a1" />
```

Comment:
I think this one is not needed anymore (`"Mr. Valid Provider" `).

Reply:
Yes, that's right!